CMakeLists.txt

```diff
@@ -587,6 +587,18 @@ if(WIN32)
   add_compile_options(/Zl) # omit default library name in .OBJ
 endif(WIN32)
 
+#------------------------------
+# Add Test Directory
+#------------------------------
+if(CLR_CMAKE_BUILD_TESTS)
+  # remove some definitions for test build
+  remove_definitions(-D_SECURE_SCL=0)
```

Comment:
These three remove_definitions are not needed anymore, the definitions are not present yet (they are set by the `include(clrdefinitions.cmake)` below)

Reply:
Thanks.  Will remove.